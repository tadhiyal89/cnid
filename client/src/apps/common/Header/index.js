import React from 'react';
import './style/index.css';

/**
 *
 * @function header
 * @export
 * @param {*} props
 * @returns it will return header html
 */
export default function header(props) {
  const { title } = props;
  return <div className="main-header"><h1 className="title-heading">{title}</h1></div>;
}
