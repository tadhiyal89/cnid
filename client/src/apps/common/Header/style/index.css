.main-header{
    display: flex;
    flex-direction: column;
    align-content: space-between;
    background-color: #fff;
    padding:10px;
    width:100%;
    position: fixed;
    z-index:8;
}   
.title-heading{
    padding:10px;
    margin:0;
    text-align:center;
}
