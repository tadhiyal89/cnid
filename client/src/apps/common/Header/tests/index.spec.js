import React from 'react';
import { shallow } from '../../../../config/enzyme';
import Header from '../index';

let wrapper;
const props ={
    title:"Landing Heading"
}
function renderShallow() {
  wrapper = shallow(<Header {...props}/>);
}


describe('Title Component', () => {

  it('#1: Snapshot testing', () => {
    renderShallow();
    expect(renderShallow).toMatchSnapshot();
  });
  it('#2: Check Title', () => {
    renderShallow();
    expect(wrapper.find('h1').text()).toBe("Landing Heading");
  });
});
