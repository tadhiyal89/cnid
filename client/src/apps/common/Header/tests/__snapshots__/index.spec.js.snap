// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Component #1: Snapshot testing 1`] = `[Function]`;
